### ISSUE TITLE
Automate https://pulp.plan.io/issues/2903

### ISSUE THREAD

**Issue by preethit** [PulpQE/pulp-smash#714]
https://pulp.plan.io/issues/2903

1. Enable Process Recycling
2. Configure the event notifier
3. Ensure (1) and (2) are both working as expected
4. Dispatch lots of tasks
5. Observe that not all events are notified

**Comment by kersommoura** [comment]
After further investigation, I was able to recreate the steps 1 to 4. However, the step 5 related to the final assertion requires an extra component not present in Pulp-Smash right now. 
It does require a web server listening for a call in case an HTTP event was triggered.
 
As per documentation:

> The HTTP notifier is used to trigger a callback to a URL when the event fires. The callback is a POST operation, and the body of the call will be the contents of the event (and thus vary by type).

`{
  "notifier_type_id" : "http",
  "notifier_config" : {
    "url" : "http://localhost/api"
  },
  "event_types" : ["repo.sync.finish", "repo.publish.finish"]
}
`

This URL will defined during process to create the event notifier.



[HTTP Norifier](https://docs.pulpproject.org/dev-guide/integration/events/http.html)

In my opinion is not suitable for automation. And this issue should be closed.

@preethit , @Ichimonji10 , please let me know if we think that we should follow a different path.

**Comment by Ichimonji10** [comment]
I agree that this isn't suitable for automation at this time. We can manually test this, though. One way to manually test it is to start a web server with `python -m http.server`, and to make Pulp call back to it.
